service/routes.py

```diff
@@ -22,3 +22,38 @@ def index():
         ),
         status.HTTP_200_OK,
     )
+
+
+@app.route("/wishlists/<int:wishlist_id>", methods=["GET"])
+def get_wishlists(wishlist_id):
+    # TODO: implement this route
+    # placeholder here so create_wishlists can return a location
+    pass
+
+
+@app.route("/wishlists", methods=["POST"])
+def create_wishlists():
+    app.logger.info("Request to create a wishlist")
+    check_content_type("application/json")
+    wishlist = Wishlist()
+    wishlist.deserialize(request.get_json())
+    wishlist.create()
+    message = wishlist.serialize()
+    location_url = url_for("get_wishlists", wishlist_id=wishlist.id, _external=True)
+
+    app.logger.info("Wishlist with ID [%s] created.", wishlist.id)
+    return make_response(
+        jsonify(message), status.HTTP_201_CREATED, {"Location": location_url}
+    )
+
+
+def check_content_type(media_type):
+    """Checks that the media type is correct"""
+    content_type = request.headers.get("Content-Type")
+    if content_type and content_type == media_type:
+        return
+    app.logger.error("Invalid Content-Type: %s", content_type)
+    abort(
+        status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
```

Comment:
i'll update the README to reflect the new ```415``` status

Reply:
Ahh thanks for catching, i just took the example tests from the prof's template repo and didn't notice the discrepancy